tox.ini

```diff
@@ -2,6 +2,8 @@
 envlist = py27,py34,pypy,py26
 [testenv]
 deps =
-  nose
   coverage
+  nose
+  ordereddict
+  unittest2
```

Comment:
@centralniak Why the new `unittest2` requirement? Are we using it for something?